src/envoy/auth/jwt.h

```diff
@@ -0,0 +1,48 @@
+#ifndef PROXY_JWT_H
+#define PROXY_JWT_H
+
+#include "openssl/evp.h"
+#include "rapidjson/document.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace Envoy {
+namespace Http {
+namespace Auth {
+
+namespace Base64url {
```

Comment:
the two namespace can be moved to `.cc` file if you don't expect other files to use it.